layouts/ucf-faq-category-classic.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+if ( ! function_exists( 'ucf_faq_category_list_display_classic_before' ) ) {
+	function ucf_faq_category_list_display_classic_before( $content, $items, $args ) {
+		ob_start();
+	?>
+		<div class="faq-category-list-wrapper">
+	<?php
+		return ob_get_clean();
+	}
+	add_filter( 'ucf_faq_category_list_display_classic_before', 'ucf_faq_category_list_display_classic_before', 10, 4 );
+}
+
+if ( ! function_exists( 'ucf_faq_category_list_display_classic_title' ) ) {
+	function ucf_faq_category_list_display_classic_title( $content, $items, $args ) {
+		$formatted_title = '';
+		if ( $title = $args['title'] ) {
+			$formatted_title = '<h2 class="ucf-faq-category-list-title">' . $title . '</h2>';
+		}
+		return $formatted_title;
+	}
+	add_filter( 'ucf_faq_category_list_display_classic_title', 'ucf_faq_category_list_display_classic_title', 10, 4 );
+}
+
+if ( ! function_exists( 'ucf_faq_category_list_display_classic' ) ) {
+	function ucf_faq_category_list_display_classic( $content, $items, $args ) {
+		ob_start();
+		if( $items ):
+			foreach( $items as $item ) :
+	?>
+				<<?php echo $args['category_element']; ?> class="ucf-faq-category-title <?php echo $args['category_class']; ?>"><?php echo $item->name; ?></<?php echo $args['category_element']; ?>>
+				<p class="ucf-faq-category-content">
```

Comment:
Just to help prevent issues with WP's content filters, you may want to consider updating `.ucf-faq-category-content` to be a div (so that, if the content filters generate extra `<p>` tags around this description value, the markup will still be valid)